crates/kubelet/src/webserver/mod.rs

```diff
@@ -4,12 +4,12 @@ use crate::provider::{NotImplementedError, Provider};
 use http::status::StatusCode;
 use http::Response;
 use hyper::Body;
+use std::convert::Infallible;
+use std::sync::Arc;
```

Comment:
This is inherited from existing code but is it worth tidying up - the doc comments in the middle of the imports?